src/Templates/MethodTemplate.cshtml

```diff
@@ -148,10 +148,14 @@ func (client @(Model.Owner)) @(Model.PreparerMethodName)(@(Model.MethodParameter
 func (client @(Model.Owner)) @(Model.SenderMethodName)(req *http.Request) (@senderRetType, error) {
 @if (Model.IsLongRunningOperation())
 {
+    var sender = "client";
+    if (Model.RegisterRP)
+    {
+        sender = "autorest.DecorateSender(client, azure.DoRetryWithRegistration(client.Client))";
+    }
```

Comment:
`DoRetryForStatusCodes` is missing when not registering RPs

Reply:
Fixed.